### PR TITLE
fixed grammar mistakes in Getting started

### PR DESCRIPTION
**Pull request by aarontorres0**

### CODE REVIEW

File: docs/getting-started/anti-phishing.md [hunk 1]
```diff
@@ -2,28 +2,28 @@
...
 
-- have the following **warning added**. This happens when the authentication fails but the sender doesn’t set any specific action. The email can be a phishing attempt and it’s important for SimpleLogin to inform you about the potential risk. Unfortunately, false positives do happen, especially when the sender incorrectly set up their DMARC. It would be beneficial to inform the sender in this case as their emails might end up in Spam everywhere.
+- have the following **warning added**. This happens when the authentication fails but the sender doesn’t take any specific action. The email can be a phishing attempt, and it’s important for SimpleLogin to inform you about the potential risk. Unfortunately, false positives do happen, especially when the sender incorrectly sets up their DMARC. It would be beneficial to inform the sender in this case, as their emails might end up in Spam everywhere.
```
Comment (nguyenkims) [inline docs/getting-started/anti-phishing.md:11]: the sender actually can't take any action but rather set a policy on what action they want the receiver to take.

**Review by nguyenkims — COMMENTED** [review]
Thanks for improving the doc! Everything looks good, except just one point about the DMARC policy just I left a comment on.

File: docs/getting-started/anti-phishing.md (hunk 1 above)
Comment (aarontorres0) [reply docs/getting-started/anti-phishing.md]: I see. Just changed it back

**Review by nguyenkims — APPROVED** [review]
LGTM!

**Pull request merged**